STEERING-COMMITTEE.md

```diff
@@ -46,6 +49,40 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+
+1. Review and approve definition of Knative Core scope.
+
+1. Only the Trademark Committee can redefine its own charter and authority over
+   Knative Trademark, conformance, and brand decisions.
+
+## Trademark Committee Composition
+
+The Trademark Committee is composed of company representatives from
+companies which have substantially contributed to Knative over the last
+year. For the 2020 year, one seat is assigned to each of Google, IBM/RedHat, and
```

Comment:
Something implicit with elections is that folks have to have some level of community recognition and (likely) trust to win an election.  This likely means those folks have sufficient context to perform the role, and presence to be approachable by the community at large.

I'd like to better understand how "company representatives" are selected.  In particular, it would be useful if there were *some* requirement of community participation, which hardly feels like it's asking a lot considering the seats are being allocated based on "substantial contribution".

Reply:
@mattmoor I think it's more important to optimize the KTC for its ability to influence the TM holder's decision making with its advice, than for its ability to participate in the community ... otherwise we could be creating a lame duck body. That doesn't mean we can't suggest that community contribution is valued in these members, but phrasing it that way sounds weak to me.

Reply:
> for its ability to influence the TM holder's decision making with its advice

Can you give an example?  My supposition was that KTC <-> mark holder would be a fairly low-touch engagement (though I agree that when it matters, it is important to be able to fulfill their purpose).  I'd also suppose that KTC <-> community would be higher-touch as features come through the WGs.

Honestly, I find the suggestion implicit in your statement concerning since if the mark holder were neutral, I wouldn't see this committee's "ability to influence" as really even a question (I may be wrong, examples of that would also be helpful).  It would be good to understand how/where this committees ability to influence Google departs from where it might be able to influence a neutral holder.

How would IBM/Redhat or VMware "optimize ... for ... ability to influence [Google]'s decision making" when picking their KTC appointees?  🤔